### PR TITLE
LRDOCS-3852 Gradle properties descriptions

### PR DESCRIPTION
**Pull request by codyhoag**
https://issues.liferay.com/browse/LRDOCS-3852

@gamerson

### CODE REVIEW

File: modules/sdk/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties [hunk 1]
```diff
@@ -48,7 +46,7 @@
 
     #
     # Set this to true to configure Liferay CDN as the default repository in the
-    # root project. The default value is false.
+    # root project. The default value is true.
     #
     #liferay.workspace.default.repository.enabled=false
```
Comment (gamerson) [inline modules/sdk/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties]: If the default value is true, then should it be listed as true in the properties file line that is commented out?

File: modules/sdk/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties [hunk 2]
```diff
@@ -70,7 +68,7 @@
 
     #
     # Set this to true to configure Liferay CDN as the default repository for
-    # module/OSGi projects. The default value is false.
+    # module/OSGi projects. The default value is true.
     #
     #liferay.workspace.modules.default.repository.enabled=false
```
Comment (gamerson) [inline modules/sdk/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties]: Same thing for here.

File: modules/sdk/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties (hunk 1 above)
Comment (codyhoag) [inline modules/sdk/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties]: @gamerson I updated this in 7a61f9aa06bd51fc7025afb5614ee758e2e267db.

**Comment by Ithildir** [comment]
Sent to https://github.com/brianchandotcom/liferay-portal/pull/50719.

**Pull request closed without merging**